x/sequencer/keeper/params.go

```diff
@@ -3,36 +3,62 @@ package keeper
 import (
 	"time"
 
+	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
 )
 
-// GetParams get all parameters as types.Params
-func (k Keeper) GetParams(ctx sdk.Context) types.Params {
-	return types.NewParams(
-		k.MinBond(ctx),
-		k.UnbondingTime(ctx),
-		k.LivenessSlashMultiplier(ctx),
-	)
+// ValidateParams is stateful validation for params
+// it checks if that UnbondingTime greater then x/rollapp's disputePeriod
+// and that the correct denom is set
+func (k Keeper) ValidateParams(ctx sdk.Context, params types.Params) error {
+	// validate unbonding time > dispute period
+	rollappParams := k.rollappKeeper.GetParams(ctx)
+	// Get the time duration of the dispute period
+	disputeDuration := time.Duration(rollappParams.DisputePeriodInBlocks) * (6 * time.Second) // dispute period duration, assuming 6 seconds per block
```

Comment:
6*time.Second
Should probably define a const somewhere since this value will pop up in other places I'm sure